meta-dts-distro/recipes-dasharo/3mdeb-secpack/3mdeb-secpack_git.bb

```diff
@@ -38,7 +39,6 @@ do_install() {
 
 addtask prepare_keyring after do_compile before do_install
 
-FILES:${PN} = " \
+FILES:${PN} += " \
     ${ROOT_HOME} \
-    ${sysconfdir} \
```

Comment:
@PLangowski, why this one have been deleted? Is this directory added to `FILES` by environment?

Reply:
Yes, it's added by poky by default https://git.yoctoproject.org/poky/tree/meta/conf/bitbake.conf#n332